test/features/execute_skos_history.feature

```diff
@@ -2,28 +2,26 @@
 # Author: Eugeniu Costetchi
 # Email: costezki.eugen@gmail.com
 
-Feature: Generate the diff of two dataset versions
+Feature: Running the skos-history diff of two dataset versions
 
   As a user,
-  I want to execute the diff of two dataset versions,
-  so that I can query the insertions and deletetions between them.
+  I want to run the skos-history script,
+  so that a diff of two dataset versions is generated.
 
-  Scenario: Main success scenario
+  Scenario: Running the skos-history
     Given a correct dataset folder structure
     And a correct configuration file
-    When the user runs the diff calculator
+    When the user runs the skos-history calculator
     Then the dataset versions are loaded into the triplestore
     And the DSV description is generated
     And the insertions and deletions graphs are created
 
-  Scenario: Incorrect config scenario
-    Given a correct dataset folder structure
-    And an incorrect configuration file
-    When the user runs the diff calculator
-    Then an error message is generated indicating the config problem
-
-  Scenario: Incorrect folder structure scenario
-    Given an incorrect dataset folder structure
-    And a correct configuration file
-    When the user runs the diff calculator
-    Then an error message is generated indicating the folder structure problem
+  Scenario Outline: Controlling the skos-history inputs
+    Given a <folder structure>
+    And a <configuration file>
+    When the user runs the skos-history calculator
+    Then an error message is generated indicating the <problem>
+    Examples:
```

Comment:
sexy

Reply:
I should have commented a few lines below so that it is understandable why the **sexy** adjective